### ISSUE TITLE
Having trouble with login

### ISSUE THREAD

**Issue by arguiot** [travis-ci/travis.rb#522]
I just installed Travis-Ci, I don't have 2nd-factor authentication activated and this is the error:
```bash
travis login
We need your GitHub login to identify you.
This information will not be sent to Travis CI, only to api.github.com.
The password will not be displayed.

Try running with --github-token or --auto if you don't want to enter your password anyway.

Username: arguiot
Password for arguiot: **********
Did not recognize your adapter specification (dlopen(/usr/local/lib/ruby/gems/2.4.0/gems/json-2.0.3/lib/json/ext/parser.bundle, 9): Library not loaded: /usr/local/opt/ruby/lib/libruby.2.4.0.dylib
  Referenced from: /usr/local/lib/ruby/gems/2.4.0/gems/json-2.0.3/lib/json/ext/parser.bundle
  Reason: image not found - /usr/local/lib/ruby/gems/2.4.0/gems/json-2.0.3/lib/json/ext/parser.bundle).
for a full error report, run travis report
```
Having any idea of what it can be and how I can fix it?🤔

**Comment by valeriy-kolodiy** [comment]
@arguiot I've managed to fix this issue by switching to **ruby-2.7.0**.

**Comment by arguiot** [comment]
@valeriy-kolodiy Alright, I’ll try. I’m just a bit upset with the TravisCi teams since I open this issue 3 years ago...
